modules/node/src/config/config.service.ts

```diff
@@ -293,32 +297,55 @@ export class ConfigService implements OnModuleInit {
     return parseInt(this.get(`INDRA_APP_CLEANUP_INTERVAL`) || "3600000");
   }
 
-  async getDefaultRebalanceProfile(
-    assetId: string = AddressZero,
-  ): Promise<RebalanceProfile | undefined> {
+  getDefaultRebalanceProfile(assetId: string = AddressZero): RebalanceProfile | undefined {
     if (assetId === AddressZero) {
+      let defaultProfileEth = {
+        collateralizeThreshold: parseEther(`0.05`),
+        target: parseEther(`0.1`),
+        reclaimThreshold: parseEther(`0.5`),
+      };
+      try {
+        const parsed = JSON.parse(this.get("INDRA_DEFAULT_REBALANCE_PROFILE_ETH"));
+        if (parsed) {
+          defaultProfileEth = {
+            collateralizeThreshold: BigNumber.from(parsed.collateralizeThreshold),
+            target: BigNumber.from(parsed.target),
+            reclaimThreshold: BigNumber.from(parsed.reclaimThreshold),
+          };
+        }
+      } catch (e) {}
       return {
         assetId: AddressZero,
         channels: [],
         id: 0,
-        collateralizeThreshold: parseEther(`0.05`),
-        target: parseEther(`0.1`),
-        reclaimThreshold: Zero,
+        ...defaultProfileEth,
       };
     }
+    let defaultProfileToken = {
+      collateralizeThreshold: parseEther(`5`),
+      target: parseEther(`20`),
+      reclaimThreshold: parseEther(`100`),
+    };
+    try {
+      defaultProfileToken = JSON.parse(this.get("INDRA_DEFAULT_REBALANCE_PROFILE_TOKEN"));
```

Comment:
what if you support the use of multiple tokens? could we make the `INDRA_DEFAULT_REBALANCE_PROFILE` an array keyed on assetId?

Reply:
I dunno, I thought the same and figured it would get complicated, so this is basically going to set it for all tokens. We can revisit if this becomes a problem. Also at that point, node operators might have to start setting rebalance profiles per channel.